sdk/servicebus/azure-servicebus/azure/servicebus/aio/management/_management_client_async.py

```diff
@@ -136,10 +136,16 @@ def from_connection_string(cls, conn_str: str, **kwargs: Any) -> "ServiceBusAdmi
         :param str conn_str: The connection string of the Service Bus Namespace.
         :rtype: ~azure.servicebus.management.aio.ServiceBusAdministrationClient
         """
-        endpoint, shared_access_key_name, shared_access_key, _ = parse_conn_str(conn_str)
+        endpoint, shared_access_key_name, shared_access_key, _, token, token_expiry = parse_conn_str(conn_str)
+        print(conn_str)
+        print(shared_access_key_name)
```

Comment:
these two `print` lines shall be removed?

Reply:
I almost want to add print checks to static analysis for when I inevitably leave debugging code in.  Thanks++